contrib/sort.py

```diff
@@ -16,34 +20,38 @@
     $ ./sort.py ~/.config/firejail/*.{profile,inc,local}
     $ sudo ./sort.py /etc/firejail/*.{profile,inc,local}
 
-Exit-Codes:
-  0: No Error; No Profile Fixed.
-  1: Error, one or more profiles were not processed correctly.
-  101: No Error; One or more profile were fixed.
+Exit Codes:
+  0: Success: No profiles needed fixing.
+  1: Error: One or more profiles could not be processed correctly.
+  101: Warning: One or more profile were fixed.
```

Comment:
I wouldn't say that this is a warning. More like a `Success,Information:`.

Reply:
@rusty-snake [on Oct 18](https://github.com/netblue30/firejail/pull/5429#discussion_r998576284):

> I wouldn't say that this is a warning. More like a `Success,Information:`.

Yeah, that makes more sense; changed to `Info:`.